docusaurus/docs/learn/quickstarts/network/help/upgrade-quickstart-network.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Upgrade a Quickstart Network
+id: upgrade-quickstart-network
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This document explains the process for backing up, and upgrading your Ziti Controller and Ziti Router(s).
+
+## Backup Existing Network
+You can certainly back up the entire Ziti Network directory (defaulted to `~/.ziti`). However, if you prefer to only 
+back up the minimum necessary files, here is how to go about doing that.
+1. Create a database backup file by running `ziti edge db snapshot` and copy that file to a backup location. The 
+backup file can be found in the same directory as the Controller's db file which is defaulted to `$ZITI_HOME/db/`. The 
+Controller's config file will also identify where the db is stored by checking the `db` section in the config.
+2. Backup the controller PKI by copying the `$ZITI_HOME/pki/` directory to a backup location
+3. Backup the controller config file, also by copying the file to a backup location
```

Comment:
"to a backup location" shows up three times. By the third time it made me think it was redundant and only needs to be mentioned once